modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -257,6 +258,123 @@ Test(diskq_truncate, test_diskq_truncate_on_push)
   _save_diskqueue(q);
 }
 
+static void
+_assert_cursors_are_at_start(LogQueue *q)
+{
+  QDisk *qdisk = ((LogQueueDisk *)q)->qdisk;
+
+  cr_assert_eq(qdisk->hdr->read_head, QDISK_RESERVED_SPACE, "Read head was not reset!");
+  cr_assert_eq(qdisk->hdr->write_head, QDISK_RESERVED_SPACE, "Write head was not reset!");
+  cr_assert_eq(qdisk->hdr->backlog_head, QDISK_RESERVED_SPACE, "Backlog head was not reset!");
+}
+
+Test(diskq_truncate, test_diskq_truncate_size_ratio_default)
+{
+  const gint truncate_size_ratio = (gint)(1100000 * 0.01);
+  const gint below_threshold_message_number = truncate_size_ratio / 134;
+  const gint above_threshold_message_number = below_threshold_message_number + 1;
+
+  LogQueue *q;
+  GString *filename = g_string_new("test_dq_truncate_size_ratio_default.rqf");
+
+  DiskQueueOptions options;
+  q = _create_reliable_diskqueue(filename->str, &options, TRUE);
+  options.truncate_size_ratio = 0.01;
```

Comment:
If this is testing the default value, the don't set it please.

Otherwise I see that it's easier from test side if you just set it, rather then get the default value after you've initialized the disk-buffer.

Reply:
I changed the order of the setting, and obtained the default value from `DiskQueueConfig`, so these should be fixed now.